nss-day-cohort-48/giffygram-altavista-giffygram
# Vc sendtoapi

#### Changes Made

For each of UserProvider, PostProvider, MessageProvider:
- added a new[Thing] function that does validation and calls post[Thing] or returns an error
- added a post[Thing] function to `dataAccess.js`   ​

Re-Rendering the page now produces some console output.

Added `testers.js` for manual API calls via the console.

#### Steps to Review

1. Checkout this branch locally.
   ```
   git fetch --all
   git checkout branchname
   ```
2. Open a new Terminal and open the giffygram repo (make sure it's named 'giffygram')
3. Run `./serve-giffygram` to serve the code.
4. Test app functionality.
  - make sure the page still loads
  - try out `makePost()` or `sendMessage()` from the console! (just dont forget to `stateChanged()`
5. View code file.
   > Confirm file modifications are present as indicated above.
   > Confirm no unused code or extraneous comments exist.
